apps/projects/app/components/Content/Issues.js

```diff
@@ -589,34 +641,55 @@ class Issues extends React.PureComponent {
                 repoId => downloadedRepos[repoId].hasNextPage
               ).length > 0
 
+            const preparedIssues = this.shapeIssues(issuesFiltered).sort(
+              currentSorter
+            )
+
             return (
               <StyledIssues>
                 {this.actionsMenu(downloadedIssues, issuesFiltered)}
                 {this.filterBar(downloadedIssues, issuesFiltered)}
+                <IssuesScrollView ref={this.setScrollElementRef}>
+                  <List
+                    height={scrollHeight}
+                    itemCount={preparedIssues.length}
+                    itemSize={92}
+                    style={{ overflow: 'overlay', paddingBottom: '30px' }}
+                  >
+                    {({ index, style }) => (
+                      <div
+                        style={{
+                          paddingRight: '30px',
+                          paddingLeft: '30px',
+                          ...style,
+                        }}
+                        key={index}
+                      >
+                        <Issue
+                          isSelected={
+                            preparedIssues[index].id in
+                            this.state.selectedIssues
+                          }
+                          {...preparedIssues[index]}
+                          onClick={this.handleIssueClick}
+                          onSelect={this.handleIssueSelection}
+                          onReviewApplication={onReviewApplication}
+                          onSubmitWork={onSubmitWork}
+                          onRequestAssignment={onRequestAssignment}
+                          onAllocateSingleBounty={
+                            this.handleAllocateSingleBounty
+                          }
+                          onUpdateBounty={this.handleUpdateBounty}
+                          onReviewWork={onReviewWork}
+                        />
+                      </div>
+                    )}
+                  </List>
+                </IssuesScrollView>
 
-                <IssuesScrollView>
+                {/* <IssuesScrollView>
```

Comment:
Remove this instead of commenting, this also orphans at least one function (showMoreIssues) so make sure to cleanup any unused functions and params along with this change.